seacatauth/authz/role/handler/role.py

```diff
@@ -26,16 +28,22 @@ def __init__(self, app, role_svc):
 		self.RoleService = role_svc
 
 		web_app = app.WebContainer.WebApp
-		web_app.router.add_get("/role", self.list_all)
-		web_app.router.add_get("/role/{tenant}", self.list)
-		web_app.router.add_get("/role/{tenant}/{role_name}", self.get)
-		web_app.router.add_post("/role/{tenant}/{role_name}", self.create)
-		web_app.router.add_delete("/role/{tenant}/{role_name}", self.delete)
-		web_app.router.add_put("/role/{tenant}/{role_name}", self.update)
+		web_app.router.add_get("/role", self.list_all_roles)
+		web_app.router.add_get("/role/*/{role_name}", self.get_global_role)
+		web_app.router.add_post("/role/*/{role_name}", self.create_global_role)
+		web_app.router.add_put("/role/*/{role_name}", self.update_global_role)
+		web_app.router.add_delete("/role/*/{role_name}", self.delete_global_role)
 
+		web_app.router.add_get("/role/{tenant}", self.list_roles)
+		web_app.router.add_get("/role/{tenant}/{role_name}", self.get_role)
+		web_app.router.add_post("/role/{tenant}/{role_name}", self.create_role)
+		web_app.router.add_put("/role/{tenant}/{role_name}", self.update_role)
+		web_app.router.add_delete("/role/{tenant}/{role_name}", self.delete_role)
```

Comment:
Reason:
Global role administration had to be separated from tenant role administration, because `*` is not a valid tenant and would not pass through ASAB Tenant Service.